src/main/resources/templates/layouts/baseLayout.html

```diff
@@ -29,5 +29,10 @@
         </main>
     </div>
     <div th:replace="fragments/footer :: footer"></div>
+    <script type="text/javascript">
+        document.body.className = ((document.body.className) ? document.body.className
+                + ' js-enabled'
+                : 'js-enabled');
+    </script>
```

Comment:
It's worth considering using jQuery to append the class to the 'body' element in this case, which would be more readable: `$('body').addClass('js-enabled');`

Reply:
Moved to cdn, PR: https://github.com/companieshouse/cdn.ch.gov.uk/pull/261